src/Test.UnitTests.Sarif.Driver/Sdk/PluginDriverCommandTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+
+using Moq;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Driver.Sdk
+{
+    public class PluginDriverCommandTests
+    {
+        [Fact]
+        public void PluginDriverCommand_ValidateInvocationPropertiesToLog_ShouldValidateParameters()
+        {
+            var sb = new StringBuilder();
+            var mockContext = new Mock<IAnalysisContext>();
+            var mockLogger = new Mock<IAnalysisLogger>();
+
+            mockContext.SetupGet(context => context.Logger).Returns(mockLogger.Object);
+            bool current = PluginDriverCommand<string>.ValidateInvocationPropertiesToLog(mockContext.Object,
+                                                                                         null);
+            current.Should().BeTrue();
+
+            current = PluginDriverCommand<string>.ValidateInvocationPropertiesToLog(mockContext.Object,
+                                                                                    Array.Empty<string>());
+            current.Should().BeTrue();
+
+            current = PluginDriverCommand<string>.ValidateInvocationPropertiesToLog(mockContext.Object,
```

Comment:
>ValidateInvocationPropertiesToLog [](http://example.com/codeflow?start=50&length=33)

I don't understand these tests. Why are you bundling all four in one unit test? There's no explanation here of why one test fails and another one passes.